fe/fe-core/src/main/java/com/starrocks/leader/UpdateProgress.java

```diff
@@ -0,0 +1,66 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.leader;
+
+import com.starrocks.common.FeConstants;
+import com.starrocks.common.util.LeaderDaemon;
+import com.starrocks.scheduler.TaskRun;
+import com.starrocks.scheduler.TaskRunManager;
+import com.starrocks.scheduler.persist.UpdateProgressInfo;
+import com.starrocks.server.GlobalStateMgr;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class UpdateProgress extends LeaderDaemon {
```

Comment:
The name UpdateProgress is too general; I can't know precisely what update it will do.
So I think you can give this class a more exact name to make others know it clearer.

Reply:
OK，I have changed the name and add more information